add doubble dash args support for custom command (#1046)
cloudposse/atmos: website/docs/core-concepts/custom-commands/custom-commands.mdx

atmos greet
```

> ## Trailing Argument

Reviewer: ```suggestion
## Trailing Arguments
```

> Atmos supports trailing arguments. This aregument is the one that is added by the user after `--`. 

Reviewer: ```suggestion
Atmos supports **positional arguments** after `--` (a standalone double-dash). The `--` itself is a delimiter that signals the end of Atmos-specific options. Anything after `--` is passed directly to the underlying command without being interpreted by Atmos.
```

For the example, adding the following to `atmos.yaml` will introduce a new `echo` command that accepts one `name` argument and also uses trailingArgs

```yaml
- name: echo
  description: "Displays a args before and after double dash."
  arguments:
    - name: name
      description: >-
          Enter your string as an argument
      required: true
      default: before double dash
  steps:
    - "echo Before Trailing Dash: {{ .Arguments.name }}"
    - "echo After Trailing Dash: {{ .TrailingArgs }}"
```
Output:

```bash

> $ atmos echo Sam -- trailing dash
> Before Trailing Dash: Sam
> After Trailing Dash: trailing dash

Reviewer: ```suggestion
$ atmos run-ansible -- -i inventory.yaml --limit web
Running: ansible-playbook site.yml -i inventory.yaml --limit web

PLAY [web] *********************************************************************

```
